### PR TITLE
Add triaging documentation to contribution guide

### PR DESCRIPTION
**Pull request by gregmuellegger**
This PR documents and summarizes our approach at triaging open tickets.

It's open for discussion. See here for the current formatted version: https://github.com/rtfd/readthedocs.org/blob/triaging-docs/docs/contribute.rst

### CODE REVIEW

File: docs/contribute.rst [hunk 1]
```diff
@@ -3,32 +3,205 @@
...
+Contributing to development
+---------------------------
 
-Tickets
--------
+If you want to deep dive and help out with development on Read the Docs, then
+first get the project installed locally according to the
+:ref:`Installation Guide <installing-read-the-docs>`. After that is done we
+suggest you have a look at tickets in our issue tracker that are labelled `Good
+First Bug`_. These are a meant to be a great way to get a smooth start and
```
Comment (agjohnson) [inline docs/contribute.rst]: 'a meant' -> 'meant'

File: docs/contribute.rst [hunk 2]
```diff
@@ -3,32 +3,205 @@
...
+*Priority: High*
+    If the issue is urgent, assign this label. In the best case also go forward to
+    resolve the ticket yourself as soon as possible.
+
+*Community Effort*
+    There are many valuable ideas in the issue tracker for future enhancements
+    of Read the Docs. Unfortunately to many for the core developers to handle
+    all of them. Therefore we assign the *Community Effort* label on all the
+    issues that we see as valid for the project but that we currently do not
+    have the resources for to work on. We encourage community members to work
```
Comment (agjohnson) [inline docs/contribute.rst]: 'for to work on' -> 'to working on' maybe?

**Comment by agjohnson** [comment]
This is great! I'm very happy with the triage process we have laid out now, and this doc does a great job of setting the tone for contributing to RTD. I provided some minor copy edits, but +1 from me otherwise

**Comment by agjohnson** [comment]
Also, while we're formalizing on this, another use case is to create tickets to be addressed in the future. I've just added a few tickets with the label `Status: blocked` with a reference to the blocking ticket to be addressed first. If this is an agreeable amendment, might as well add that here as well.

File: docs/contribute.rst [hunk 3]
```diff
@@ -3,32 +3,205 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Additionally to the labels already involved in the section above, we have a
+few more at hand to further categorize issues.
+
+*Priority: High*
+    If the issue is urgent, assign this label. In the best case also go forward to
+    resolve the ticket yourself as soon as possible.
+
+*Community Effort*
```
Comment (destroyerofbuilds) [inline docs/contribute.rst]: I'm not sure I really understand this label. My assumption is that all none assigned issues could be taken on by the community? Maybe the `accepted` label is the label that community members should look at for determining whether an issue is valid for working on?

**Comment by destroyerofbuilds** [comment]
There's no mention of what `Status: accepted` means.

File: docs/contribute.rst (hunk 3 above)
Comment (agjohnson) [inline docs/contribute.rst]: Community Effort is a label for issues that we will not be focusing our time on. If someone from the community wants to take this on, that is fine. But we are setting the expectation that tickets under this label are not something they should expect without putting in effort.

Status: accepted implies we have interest in taking on the ticket at some point.

**Comment by agjohnson** [comment]
Status: accepted is not valid anymore, as all tickets will be labeled after triaging.

**Comment by ericholscher** [comment]
It might be useful to have a full listing of "Label -> What it means" in this doc, but I believe this is a great place to start, and we should get it merged soon.

**Comment by gregmuellegger** [comment]
I reviewed all tickets based on the new triage process that had the label "Status: accepted". All tickets were re-assigned appropriate labels and no open ticket with "Status: accepted" was left. So I went and deleted this label to avoid further confusion about what it means.

**Comment by gregmuellegger** [comment]
I added documentation about all the labels in the issue tracker.

**Comment by agjohnson** [comment]
LGTM with typo fix

**Comment by gregmuellegger** [comment]
@agjohnson Fixed.

**Comment by ericholscher** [comment]
:fireworks:

**Pull request merged**